README.md

```diff
@@ -1 +1,148 @@
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+- **Ruby**:
+
+### Install
+
+To install on Mac Os run following commands:
+
+```sh
+ brew install rbenv ruby-build
+# Add rbenv to bash so that it loads every time you open a terminal
+echo 'if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi' >> ~/.bash_profile
+source ~/.bash_profile
+
+# Install Ruby
+rbenv install 3.0.1
+rbenv global 3.0.1
+ruby -v
+```
+Installing Ruby on Windows is a little more difficult than installing it on another OS. We would recommend using [WSL](https://learn.microsoft.com/en-us/windows/wsl/about), but you can also try to install Ruby directly on your OS with [rubyinstaller](https://rubyinstaller.org/).
+
+Note:
+`WSL only works on 64-bit installations of Windows.`
+
+If you are using a 64-bit version of Windows 10, we recommend following [this](https://gorails.com/setup/windows/10) article to install Ruby.
+
+If you can not use WSL then you should follow these steps:
+
+- Download the last version of [RubyInstaller](https://rubyinstaller.org/downloads/).
+- Run RubyInstaller and follow the steps described [here](https://stackify.com/install-ruby-on-windows-everything-you-need-to-get-going/).
+
+If you are familiar with [docker](https://www.docker.com/), you could also use it.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:MosDevx/Ruby-Enumerable.git
+```
+
+### Usage
+
+If you have ruby installed in your PC the simply start IRB(**Interactive Ruby Shell**) on terminal using:
+
+```sh
+  $ irb
+
+```
+
+Thats it now you can execute your program on termonal:)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Shakir-Hussain**
+
+- GitHub: [@Shakir-Hussain12](https://github.com/Shakir-Hussain12)
+- Twitter: [@S_Hussain_99](https://twitter.com/S_Hussain_99)
+- LinkedIn: [shakir-ussain99](https://www.linkedin.com/in/shakir-hussain99/)
+
+👤 **Moses Wamae**
+
+- GitHub: [@mosdevx](https://github.com/mosdevx)
+- Twitter: [@moseswamae7](https://twitter.com/moseswamae7)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/moses-wamae-a13a67244)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Live Demo]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+To add an [issue](https://github.com/MosDevx/Enumerable-Ruby/issues)
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
The link to the license file is broken since you are making reference `./MIT.md` but the file is named `LICENSE`. Kindly fix this error to make your readme file more proffessional :+1: